dialogs/MapDiff.cpp

```diff
@@ -0,0 +1,60 @@
+#include "dialogs/MapDiff.h"
+
+#include <QLabel>
+#include <QDialogButtonBox>
+#include <QHBoxLayout>
+#include <QPushButton>
+
+using namespace ui;
+
+CMapDiff::CMapDiff(QWidget* pParent): 
+	QDialog(pParent)
+{
+	this->setWindowTitle(tr("Map Diff"));
+
+	auto pDialogLayout = new QGridLayout(this);
+	pDialogLayout->setAlignment(Qt::AlignLeft);
+
+    auto pFindLabel = new QLabel(tr("Map to Diff Against "), this);
+	auto pOptionLabel = new QLabel(tr("Diff Options"), this);
```

Comment:
The original Hammer dialog is bad, maybe we could remove this label since there's only one option below it
I doubt anyone will mind as this is not used very often at all

Reply:
Just do it when we integrate into hammer